portal-api/src/external-api/number-generator.js

```diff
@@ -10,19 +10,16 @@ const headers = {
   'x-api-key': EXTERNAL_API_KEY,
 };
 
-const create = async ({ dealType, entityType, entityId, dealId, user }) => {
+const get = async ({ entityType, dealId }) => {
```

Comment:
Could we call it `getGeneratedId` or `getIdForEntity`/`getNumGenId` (whatever you think describes this best!)